src/GAP/gap_to_oscar.jl

```diff
@@ -164,6 +71,14 @@ GAP.gap_to_julia(::Type{QQAbFieldElem}, a::GapInt; recursive::Bool = true) = QQA
 
 (::QQAbField)(a::GapObj) = QQAbFieldElem(a)
 
+##
+## matrix conversion
+##
+
+function __ensure_gap_matrix(obj::GapObj)
+    @req GAPWrap.IsMatrixOrMatrixObj(obj) "<obj> is not a GAP matrix"
+end
```

Comment:
Why is this function needed here?
(One call of the function is still present in the file, can't we call `GAP.NemoExt.__ensure_gap_matrix` there?)